include/EVT/io/CAN.hpp

```diff
@@ -25,6 +25,16 @@ enum class Pin;
  */
 class CAN {
 public:
+    /**
+     * Represents potential errors that may take place when using the CAN
+     * interface.
+     */
+    enum class CANStatus {
+        TIMEOUT = 0,
```

Comment:
Is there any particular reason you chose these values for the enum? Isn't 0 the standard value for a successful status? In most cases, this shouldn't matter, but if you set okay to 0, you could use the enum as a boolean to have a simpler if statement to check for success (I think). I'm not certain C++ would do the implicit casting to check if the enum value is zero or nonzero to check truth or falsity, so this might be irrelevant.

Reply:
That's a good point, I'll change the numeric representation